addons/a11y/src/preset/index.ts

```diff
@@ -0,0 +1,12 @@
+export function managerEntries(entry: any[] = []) {
+  return [...entry, require.resolve('./register')];
+}
+
+export function config(entry: any[] = [], { addDecorator = true } = {}) {
+  return [
+    ...entry,
+    require.resolve('./a11yRunner'),
+    require.resolve('./a11yHighlight'),
+    addDecorator && require.resolve('./addDecorator'),
```

Comment:
@tooppaaa is this line needed? i think this will automatically add the decorator, but we only need to export it for users who are using it manually?

Reply:
You are absolutely right. I also made a mistake with resolving files from the wrong directory..